x-pack/plugins/index_lifecycle_management/public/application/sections/policy_list/policy_flyout/components/delete_searchable_snapshot.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { SerializedDeletePhase } from '../../../../../../common/types';
+import { i18nTexts } from '../../../edit_policy/i18n_texts';
+import { ActionDescription } from './action_description';
+import type { ActionComponentProps } from './types';
+
+export const DeleteSearchableSnapshot = ({ phase, phases }: ActionComponentProps) => {
+  const phaseConfig = phases[phase];
+  const deleteSearchableSnapshot = (phaseConfig as SerializedDeletePhase)?.actions.delete
+    ?.delete_searchable_snapshot;
+  return deleteSearchableSnapshot ? (
```

Comment:
If the user sets this to `false` it won't be displayed in the flyout, was this intentional?
I wonder if it wouldn't be better to always display the label and display `Yes`/`No` below, wdyt?

Reply:
Oh yes, I was debating about this description myself. Thanks a lot for the suggestion, updated in https://github.com/elastic/kibana/pull/186955/commits/ebde681943996596b08b38ba81e3ff7a1d1de4b7